smart-contracts/contracts/Unlock.sol

```diff
@@ -768,6 +769,32 @@ contract Unlock is UnlockInitializable, UnlockOwnable {
     return globalTokenSymbol;
   }
 
+  // TODO: maybe better to pass bytes here if we want to change it later?
```

Comment:
I am not sure we would ever actually pass something here? If we need data, we can hardcode it in Unlock or PublicLock

Reply:
Yes, making everything hardcoded and public will make things safer. Currently we pass `_previousUnlockAddress` as arg so we avoid adding a signature (saves on contract size) but I will be safer to fetch the value from `PublicLock`.